db_scripts/db_restore

```diff
@@ -0,0 +1,25 @@
+#! /bin/bash
+
+if [ -z $1 ]
+	then
+		echo "No SQL script file"
+		exit 1
+fi
+
+NAME=${2:-postgres}
+CONTAINER=$(docker ps --format "{{.Names}}" | grep "$NAME")
+if [ -z "$CONTAINER" ]
+	then
+		echo "Container not found"
+		exit 1
+fi
+
+#need to do this in case db dump file has a colon in it
+cp $1 tmp.sql
+docker cp tmp.sql "$CONTAINER:/"
```

Comment:
rather than making a tempfile, you can pass it in via stdin:
```
docker cp - "$CONTAINER:/tmp.sql" <"$1"
```

Reply:
You can only do that with tar files. Tarring and untarring would be more of a hassle that creating a tempfile.